instance/api/instance.py

```diff
@@ -47,7 +47,9 @@ def provision(self, request, pk=None):
         Start the (re-)provisioning of an instance
         """
         instance = self.get_object()
-        if instance.status not in (instance.EMPTY, instance.READY):
+        if instance.status not in (instance.EMPTY,
+                                   instance.READY,
+                                   instance.ERROR):
```

Comment:
@chromano Wouldn't it be better to check on `instance.progress` instead here? Ie:

```
if instance.progress is not instance.PROGRESS_RUNNING:
```